src/v/raft/persisted_stm.cc

```diff
@@ -80,8 +80,10 @@ persisted_stm_base<BaseT, T>::load_local_snapshot() {
 }
 template<typename BaseT, supported_stm_snapshot T>
 ss::future<> persisted_stm_base<BaseT, T>::stop() {
+    vlog(_log.debug, "Stopping");
     co_await raft::state_machine_base::stop();
     co_await _gate.close();
+    vlog(_log.debug, "Stopped");
 }
```

Comment:
do these need to be at info level to be effective?

Reply:
not for the current issue we are debugging (raft hang at shutdown) that reproduces in ducktape.

Probably a bit chatty at info. (`n` times per replica whenever it shuts down).

Reply:
ah yeh makes sense